chainstate/src/detail/error.rs

```diff
@@ -104,8 +104,28 @@ pub enum CheckBlockTransactionsError {
     DuplicateInputInTransaction(Id<Transaction>, Id<Block>),
     #[error("Duplicate input in block")]
     DuplicateInputInBlock(Id<Block>),
-    #[error("Duplicate transaction found in block")]
+    #[error("Duplicate transaction {0} found in block {1}")]
     DuplicatedTransactionInBlock(Id<Transaction>, Id<Block>),
+    #[error("Incorrect issue transaction {0} in block {1}")]
+    TokenIssueFail(Id<Transaction>, Id<Block>),
+    #[error("Incorrect transfer transaction {0} in block {1}")]
+    TokenTransferFail(Id<Transaction>, Id<Block>),
+    #[error("Too many token issues in transaction {0} in block {1}")]
+    TooManyTokenIssues(Id<Transaction>, Id<Block>),
```

Comment:
Is that only me, or is it a little bit weird to have the "issue" verb in plural?.. I'm not a native speaker, so it somewhat strange for me to complain. 🙃 I have checked [here](https://www.britannica.com/dictionary/issue) and it says that it should be singular:
>  [singular] : the act of officially making something available or giving something to people to be used : the act of issuing something 

Anyway, personally I would say `TooManyTokensIssued`.

https://www.britannica.com/dictionary/issue

Reply:
I would even prefer to call it `MultipleTokenIssuanceInTransaction`.

Reply:
Fixed

Reply:
"Too many token issues in transaction {0} in block {1}"
The issue exists here too. "issued" should be "issued" and "token" should be "tokens"

Reply:
@muursh `Too many tokens issued in transaction {0} in block {1}` ?

Reply:
Yep

Reply:
@muursh thanks, fixed!